spk/duply/Makefile

```diff
@@ -0,0 +1,26 @@
+SPK_NAME = duply
+SPK_VERS = 1.11
+SPK_REV = 5
+
+SPK_ICON = src/duply.png
+
+DEPENDS = cross/bash cross/$(SPK_NAME)
```

Comment:
I do believe duply works with busybox as well, so there might not be a need to add bash as a dependency. Might be worth checking out.